docs/buildah-containers.md

```diff
@@ -24,6 +24,11 @@ Do not truncate IDs in output.
 
 Displays only the container IDs.
 
+**--all, -a**
+
+Also list information about containers which don't appear to be buildah working
```

Comment:
Nit. Remove Also.

`List information about all containers including containers images which were not created by buildah.`